tox.ini

```diff
@@ -8,32 +8,31 @@
 
 [tox]
 # PyPy configuration (on Linux/OSX):
-# - /usr/local/bin/pypy -> /opt/pypy2.7-v7.3.0-osx64/bin/pypy
 # - /usr/local/bin/pypy3 -> /opt/pypy3.6-v7.3.0-osx64/bin/pypy3
 envlist =
-    py{27,34,35,36,37,38,39,310}-wrapt{1.10,1.11,1.12,1.13}
-    pypy, pypy3
+    py{37,38,39,310}-wrapt{1.10,1.11,1.12,1.13,1.14}
+    py{311,312}-wrapt{1.14}
```

Comment:
Do we need to test five versions of wrapt? 1.10 is from 2017, pretty old by now.

If we're adding two, shall we remove two?

Reply:
Support table:

|Wrapt version°|Pythons supported|
|---|---|
|[1.10.x](https://pypi.org/project/wrapt/1.10.11/)|2.6, 2.7, 3.3, 3.4, 3.5, 3.6|
|[1.11.x](https://pypi.org/project/wrapt/1.11.2/)|2.6, 2.7, 3.3, 3.4, 3.5, 3.6, 3.7|
|[1.12.x](https://pypi.org/project/wrapt/1.12.1/)|2.7, 3.4, 3.5, 3.6, 3.7, 3.8|
|[1.13.x](https://pypi.org/project/wrapt/1.13.3/)|2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10|
|[1.14.x](https://pypi.org/project/wrapt/1.14.1/)|2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11|
|[1.15.x rc](https://pypi.org/project/wrapt/1.15.0rc1/)|2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11|


It looks very safe to drop 1.10 because none of its supported versions would be supported in the next Deprecated release if this PR is merged. I'd call it safe enough to drop 1.11-1.12 and probably even 1.13, tbh.

°: latest in series

Reply:
I'd say let's keep the wrapt dep and testing at 1.10+ for this PR and if I have time this week (long shot), I can do the work to eliminate 1.10-1.13 to settle on 1.14+.